internal/kuberesource/mutators.go

```diff
@@ -301,6 +303,92 @@ func AddDmesg(resources []any) []any {
 	return out
 }
 
+// AddImageStore adds a PersistentVolumeClaim, a pod volume for it, and a holder container
+// that attaches the PVC's block device so it appears in the pod VM.
+func AddImageStore(resources []any) []any {
+	getConfigs := func(storeSize string) (*applycorev1.ContainerApplyConfiguration, *applycorev1.VolumeApplyConfiguration) {
+		holderContainer := Container().
+			WithName("pvc-holder").
+			WithImage("ghcr.io/edgelesssys/bash@sha256:cabc70d68e38584052cff2c271748a0506b47069ebbd3d26096478524e9b270b").
+			WithCommand("/usr/local/bin/bash", "-c", "sleep infinity").
+			WithRestartPolicy(corev1.ContainerRestartPolicyAlways).
+			WithSecurityContext(SecurityContext().
+				WithPrivileged(true).SecurityContextApplyConfiguration).
+			WithResources(ResourceRequirements().
+				WithMemoryLimitAndRequest(50),
+			).
+			WithVolumeDevices(
+				applycorev1.VolumeDevice().
+					WithDevicePath("/dev/trusted_store").
```

Comment:
Is that path required by kata? Otherwise we should change it to image store.

Reply:
Yes, it's their magic device they check for.

I suppose I could include changing its name in the patch, if you'd prefer that?